### PR TITLE
fix: overflow on public squad post page and upvote modal

### PR DESCRIPTION
**Pull request by omBratteng**
### Describe what this PR does
- fix overflow issue in upvoted modal
- fix overflow issue on public squad post page and modal

<table>
<tr>
 <td>Before
 <td>After
<tr>
 <td>
  
  
  
  
 <td>
  
  
  
  
</table>

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]
Perhaps we should add some user with super long handles to our seeds 👀👀👀 will help identify early

**Comment by omBratteng** [comment]
Good idea, I'll note it down to add it.
Already found another edge case with really long usernames in public squad posts 😬

File: packages/shared/src/components/post/SquadPostContent.tsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ function SquadPostContent({
       >
         <div
           className={classNames(
-            'relative px-4 tablet:px-8',
+            'relative max-w-[42.5rem] px-4 tablet:px-8',
```
Comment (omBratteng) [inline packages/shared/src/components/post/SquadPostContent.tsx:98]: I took `42.5rem` as that is the width of `pageMaxWidth`

Comment (rebelchris) [reply packages/shared/src/components/post/SquadPostContent.tsx:98]: Hmm why do we need it?
Cant it just set to full?

Comment (omBratteng) [reply packages/shared/src/components/post/SquadPostContent.tsx:98]: No, full doesn't work 😞

Comment (omBratteng) [reply packages/shared/src/components/post/SquadPostContent.tsx:98]: The parent div already have a `!w-[42.5rem]` applied to it, but when it is a squad post, it also gets `flex-1` added, which then disregards the width of the element.

Comment (omBratteng) [reply packages/shared/src/components/post/SquadPostContent.tsx]: Playing with it some more, setting `min-w-0` helped too, and it fixed another overflow issue on smaller screens.

Comment (rebelchris) [reply packages/shared/src/components/post/SquadPostContent.tsx]: Lol we might have to start applying that mind w fix on *

**Review by rebelchris — APPROVED** [review]

**Review by nensidosari — APPROVED** [review]

**Pull request merged**